executor/runtime/docker/docker_config.go

```diff
@@ -166,6 +166,22 @@ func shouldStartSpectatord(cfg *config.Config, c runtimeTypes.Container) bool {
 	return true
 }
 
+func shouldStartAtlasd(cfg *config.Config, c runtimeTypes.Container) bool {
+	enabled := cfg.ContainerAtlasd
+	if !enabled {
```

Comment:
What do you think about

```
	if cfg.AtlasdServiceImage == "" {
		return false
	}
        return cfg.ContainerAtlasd
```

Reply:
@solarkennedy and I decided to clean this up in a different pull request.